lib/libutils/ext/trace.c

```diff
@@ -53,12 +53,13 @@ int trace_get_level(void)
 
 static const char *trace_level_to_string(int level, bool level_ok)
 {
-	static const char lvl_strs[][4] = {
-		"UKN", "ERR", "INF", "DBG", "FLW" };
+	static const char lvl_strs[][9] = {
+		"UNKNOWN:", "ERROR:  ", "INFO:   ", "DEBUG:  ",
```

Comment:
OK, then what about replacing the new variable introduced in the first patch with a level-dependent setting. Such as: `CFG_MSG_LONG_PREFIX_THRESHOLD=3`. Meaning that for all messages of level 3 (debug) and higher, we get a long prefix (severity + thread id + component + function + line) otherwise a short prefix (severity + component only).